distribution/lib/Standard/Test/0.0.0-dev/src/Faker.enso

```diff
@@ -0,0 +1,39 @@
+from Standard.Base import all
+
+polyglot java import java.util.Random
+polyglot java import org.enso.base.Text_Utils
+
+upper_case_letters = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".utf_8
+numbers = "0123456789".utf_8
+
+## Creates a random number generator which can be used for creating test values.
+
+   Arguments:
+   - seed: Optional seed value to make the sequence deterministic
+make_generator : Integer -> Random
+make_generator (seed = 0) =
+    if seed == 0 then Random.new else Random.new seed
+
+
+## Creates a random string based on a template and random number generator.
+
+   Arguments:
+   - template: Vector of character arrays that represent the possible
+     characters for each letter.
+   - generator: Random number generator
+
+   > Examples:
+     Creates a fake UK National Insurance number:
+
+            l = "ABCEGHJKLMNOPRSTWXYZ".utf_8
+            n = "0123456789".utf_8
+            s = "ABCDFMP ".utf_8
```

Comment:
I'm not convinced why we require here to provide UTF-8 bytes? Cannot this just be either:
- a Text template - where each character from the text can be randomly chosen,
- or a Vector of Texts (since we don't have a separate character class yet) where each element can be chosen at random?
Using bytes seems very low-level and adds cognitive load for the user to remember to convert to bytes.
It can also be error-prone - if the user does `template = ["my_string".utf_8]` and `my_string` includes some special character which is represented in UTF-8 by multiple bytes, the random generator will split such a character into codepoints and yield some pretty weird results.

Reply:
Added pivotal ticket to look at this #181281405